src/app/_components/matches/MatchWindowClient.tsx

```diff
@@ -0,0 +1,104 @@
+"use client";
+
+import MatchWindow from "~/app/_components/matches/MatchWindow";
+import type { Match, User } from "@prisma/client";
+import { api as reactApi } from "~/trpc/react";
```

Comment:
I prefer the use of api but if you prefer to rename 'clientApi' instead of 'reactApi'

Reply:
סבבה